conf/application.conf

```diff
@@ -255,6 +265,11 @@ Prod {
         host = enrolment-store-proxy.protected.mdtp
         port = 443
       }
+      enrolment-store-proxy {
```

Comment:
`agent-assurance` rather than `enrolment-store-proxy`

Reply:
good catch - fixed it